Sources/FluentMongoDriver/MongoDB+Transaction.swift

```diff
@@ -21,7 +22,7 @@ extension FluentMongoDatabase {
                 return transactionDatabase.abort().flatMapThrowing { _ in
                     throw error
                 }
-            }
+            }.hop(to: context.eventLoop)
```

Comment:
@kacperk Wouldn't this be the only thing that's needed?

Reply:
@siemensikkema from what I was testing it is not enough - it crashes on first query inside of transaction. Without new context for transaction, database object will have different event loop on raw database object and different in context.

Reply:
Fair enough 🙂 Would you be able to write a test for this fix?

Reply:
@siemensikkema is there any documentation telling how to setup test environment?